src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Queries/ProcessAllApprenticeships.cs

```diff
@@ -8,6 +9,14 @@ namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries
 {
     public class ProcessAllApprenticeships : ICosmosDbQuery<None>
     {
+        public Expression<Func<Apprenticeship, bool>> Predicate { get; set; }
+        public int? MaxBatchSize { get; set; }
         public Func<IReadOnlyCollection<Apprenticeship>, Task> ProcessChunk { get; set; }
+
+        public ProcessAllApprenticeships(Expression<Func<Apprenticeship, bool>> predicate = null, int? maxBatchSize = null)
```

Comment:
Nit: we don't typically have constructors for these query types (+ it's not clear why there's no `ProcessChunk` parameter)

Reply:
done, that was unused, not sure how it snuck in